modules/util/portal-tools-soy-builder/build.gradle

```diff
@@ -15,8 +15,8 @@ configurations {
 }
 
 dependencies {
-	api group: "com.google.template", name: "soy", version: "2019-03-07"
```

Comment:
Is the update to the `soy` dependency necessary for this update? If not, let's leave it off.